### ISSUE TITLE
Parser error if final line has a comment

### ISSUE THREAD

**Issue by zofrex** [avakar/pytoml#17]
pytoml doesn't seem to like it if the very last line has a comment on (and there is no blank line at the end of the file).

Parses ok:
```
a=1 #comment

```

Errors:
```
a=1 #comment
```

Example:
```
>>> import pytoml as toml
>>> toml.loads('[table] #comment\n')
{u'table': {}}
>>> toml.loads('[table] #comment')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/site-packages/pytoml/parser.py", line 23, in loads
    ast = _p_toml(src)
  File "/usr/local/lib/python2.7/site-packages/pytoml/parser.py", line 344, in _p_toml
    s.expect_eof()
  File "/usr/local/lib/python2.7/site-packages/pytoml/parser.py", line 124, in expect_eof
    return self._expect(self.consume_eof())
  File "/usr/local/lib/python2.7/site-packages/pytoml/parser.py", line 164, in _expect
    raise TomlError('msg', self._pos[0], self._pos[1], self._filename)
pytoml.core.TomlError: <string>(1, 9): msg
>>> import pkg_resources; pkg_resources.get_distribution("pytoml").version
'0.1.8'
```

**Comment by avakar** [comment]
Thank you for the report, I'll look into it!

**Comment by avakar** [comment]
I released the fix as pytoml-0.1.10, thanks again! You can do `pip install pytoml --upgrade` to get it.

**Comment by zofrex** [comment]
Nice, thank you!
